src/main/webapp/WEB-INF/tags/keyboardShortcutModal.tag

```diff
@@ -819,6 +819,18 @@
                             </tr>
                         </table>
                     </li>
+                    <li>
+                        <table>
+                            <tr>
+                                <td class="kbd">
+                                    <span>shift</span>+<span>h</span>
+                                </td>
+                                <td class="kbdtext">
+                                    toggle visibility of the previous segmentation
```

Comment:
Shortcut description should follow capitalization standards of other shortcut descriptions

Reply:
Shortcut should also be added to all Tabs where the shortcut is available (in this case `Edit` and `Segment`)